packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -191,6 +191,13 @@ class ImagePicker {
   /// the front or rear camera should be opened, this function is not guaranteed
   /// to work on an Android device.
   ///
+  /// Use `forceFullMetadata` (defaults to `true`) to control, how much additional information the plugin tries to get on iOS.
```

Comment:
Nit: This shouldn't specifically say it's for iOS, as it might apply to other platforms in the future. I would remove iOS here, and add it to the sentence below (e.g., "... which may require extra permission requests on some platforms, such as iOS").